preview/galleon-local/src/main/resources/modules/system/layers/base/jakarta/persistence/api/main/module.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright The WildFly Authors
+  ~ SPDX-License-Identifier: Apache-2.0
+  -->
+
+<module xmlns="urn:jboss:module:1.9" name="jakarta.persistence.api">
+
+    <resources>
+        <artifact name="${jakarta.persistence:jakarta.persistence-api}"/>
+    </resources>
+
+    <dependencies>
+        <!-- java.instrument export due to jdeps result -->
+        <module name="java.instrument" export="true"/>
+        <module name="java.logging"/>
+        <!-- PersistenceUnitInfo needs javax.sql.DataSource -->
+        <module name="java.sql" export="true"/>
```

Comment:
Why do we export this?

Reply:
Persistence SPEC API needs the  JDBC classes from the JDK like javax.sql.DataSource in https://jakarta.ee/specifications/persistence/3.2/apidocs/jakarta.persistence/jakarta/persistence/spi/persistenceunitinfo

Reply:
So we export it so that applications will have access to DataSource and friends.